src/Middleware/src/Headstart.Common/Models/Headstart/HSOrder.cs

```diff
@@ -112,24 +108,6 @@ public class OrderDetails
         public IList<OrderApproval> Approvals { get; set; }
     }
 
-    public class ClaimsSummary
-    {
-        public bool HasClaims { get; set; }
-
-        public bool HasUnresolvedClaims { get; set; }
-
-        public List<ClaimResolutionStatuses> Resolutions { get; set; }
-    }
-
-    public class ClaimResolutionStatuses
-    {
-        public string LineItemID { get; set; }
-
-        public string RMANumber { get; set; }
```

Comment:
I was only able to determine that RMAs were managed at the lineitem level. It may have been refactored from the order level at some point.
I found no evidence that any of the properties removed were being used by the middleware or the UI apps that both performed read and write operations, therefore deemed redundant for MVP.